lib/membrane_flv_plugin/muxer.ex

```diff
@@ -12,34 +12,38 @@ defmodule Membrane.FLV.Muxer do
   use Membrane.Filter
 
   require Membrane.H264
-  alias Membrane.{AAC, Buffer, FLV, H264, RemoteStream}
+  alias Membrane.{AAC, Buffer, FLV, H264, RemoteStream, TimestampQueue}
   alias Membrane.FLV.{Header, Packet, Serializer}
 
   def_input_pad :audio,
     availability: :on_request,
-    accepted_format: %AAC{encapsulation: :none, config: {:audio_specific_config, _config}},
-    flow_control: :manual,
-    demand_unit: :buffers
+    accepted_format: %AAC{encapsulation: :none, config: {:audio_specific_config, _config}}
 
   def_input_pad :video,
     availability: :on_request,
-    accepted_format: %H264{stream_structure: structure} when H264.is_avc(structure),
-    flow_control: :manual,
-    demand_unit: :buffers
+    accepted_format: %H264{stream_structure: structure} when H264.is_avc(structure)
 
   def_output_pad :output,
     availability: :always,
-    accepted_format: %Membrane.RemoteStream{content_format: FLV},
-    flow_control: :manual
+    accepted_format: %Membrane.RemoteStream{content_format: FLV}
 
   @impl true
   def handle_init(_ctx, _opts) do
+    queue =
+      TimestampQueue.new(
+        pause_demand_boundary: Membrane.Time.milliseconds(100),
```

Comment:
seems like we should have a default for this

Reply:
default is `:infinity`

Reply:
Problematic thing is that boundary may be `pos_integer()` or `Membrane.Time.t()` depending on the `pause_demand_boundary_unit` value (`:buffers`, `:bytes` or `:time`)

Reply:
Maybe `pause_demand_boundary: {unit, value}` would be better then?